.gitignore

```diff
@@ -7,3 +7,4 @@ regenmaschine.egg-info
 coverage.xml
 poetry.lock
 tags
+dist/
```

Comment:
Let's remove this, too, since my publishing process doesn't create a local `dist/`.

Reply:
Done.